ui/desktop/app/components/session/terminal-tabs/index.js

```diff
@@ -16,32 +24,60 @@ export default class SessionTerminalTabsComponent extends Component {
       return;
     }
 
-    const xterm = new Terminal();
+    const xterm = new Terminal(terminalOptions);
     this.terminal = xterm;
     const fitAddon = new FitAddon();
     const termContainer = document.getElementById(`terminal-container`);
 
-    // TODO: Do we need the fit addon?
     xterm.loadAddon(fitAddon);
     xterm.open(termContainer);
     fitAddon.fit();
+    xterm.focus();
 
     // Generate a UUID to have the terminal handlers be unique
     const id = uuidv4();
+    this.id = id;
```

Comment:
I don't see `this.id` being used anywhere else? Is it needed or potentially refactor this to `this.id = uuidv4()`?

Reply:
It's for my next PR to kill the terminals